Identity/GraphAPI/GraphAPI.psm1

```diff
@@ -64,9 +64,13 @@ function Initialize-GraphEnvironment
 
     if ($AdfsFqdn)
```

Comment:
why did you make these changes? Please refrain from editing this file, it is used in the product and copied here periodically

Reply:
The script is incorrect for ADFS. If you run it in the ADFS env, then you will get the error. Our deployment script only goes AAD code path.

- "$Environment = 'ADFS'" is the runtime error, $Environment cannot be assigned as ADFS value, because ADFS is not in value set [ValidateSet('AzureCloud', 'AzureChinaCloud', 'AzureUSGovernment', 'AzureGermanCloud')].'

- If clear the applications with below code, then cannot signin ADFS, "$graphEnvironmentTemplate.Applications = [pscustomobject]@{}"